### DISCUSSION TITLE
Welcome to Gridap.jl Discussions!

### DISCUSSION THREAD

**Discussion by santiagobadia** [gridap/Gridap.jl#497] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by JCHostos** [comment]
Hi everyone,

I am working with Gridap.jl and generated a simple structured mesh using the following code:

domain = (0,1,0,1)
partition = (20,20)
model = CartesianDiscreteModel(domain, partition)

This creates a structured 20x20 quadrilateral mesh in the domain [0,1]×[0,1]. Now, I would like to modify the structure "model" to densify the mesh towards the edges using the following transformation:

(xn,yn)= 0.5 + 0.5 * tanh(eps * (2 * (x,y) - 1)) / tanh(eps)

where (xn,yn) are the new nodal coordinates after applying the transformation to the original coordinates (x,y), epsilon is the densification parameter.

​My Question:

How can I modify the model structure in Gridap to change only the nodal coordinates according to this transformation, while keeping the connectivity and other aspects of the mesh unchanged?. I adding pictures of what I want for clarity:

I want to past from this:
![Uniform](https://github.com/user-attachments/assets/13450b1c-fad2-4d06-a4bd-b39ee70d24e8)

to this:
![Densified](https://github.com/user-attachments/assets/9e49d6ad-c43d-4bd0-921b-44d96aab6380)
